pdc/apps/auth/views.py

```diff
@@ -134,6 +140,128 @@ def user_profile(request):
     return render(request, 'user_profile.html', context)
 
 
+def get_users_and_groups(resource_permission):
+    # get all groups
+    members = dict()
+    groups_list = []
+    try:
+        group_resource_permission_list = get_list_or_404(models.GroupResourcePermission,
+                                                         resource_permission=resource_permission)
+        groups_list = [str(obj.group.name) for obj in group_resource_permission_list]
+    except Http404:
+        pass
+
+    # get all users
+    users_list = []
+    users = models.User.objects.all()
+    for user in users:
+        uer_group_list = [group.name for group in user.groups.all()]
+        if list(set(groups_list).intersection(set(uer_group_list))):
+            users_list.append(str(user.username))
+
```

Comment:
Is above lines from line 155 same as something like following? Please help me check it,
```
users_set = { str(user.username) for user in get_user_model().objects.filter(groups__name__in=groups_list)}
users_list = list(users_set)
```

I think current implementation doesn't improve too much. Need to access DB number of users times.